p2p/ipld/plugin/nodes/nodes.go

```diff
@@ -372,7 +435,7 @@ func (l nmtLeafNode) Size() (uint64, error) {
 	return 0, nil
 }
 
-func cidFromNamespacedSha256(namespacedHash []byte) (cid.Cid, error) {
+func CidFromNamespacedSha256(namespacedHash []byte) (cid.Cid, error) {
```

Comment:
Nit: can we add a short comment now that this a public method?

Reply:
What kind of applications are expected for it outside this package?

Reply:
The DA header contains the raw namespaced hashes without the CID overhead (row and column roots). At least currently. So everything that gets a DA header and want to get some data from the NMT, will need to transform namespacedHash to a CID before it can dagGet anything related to that row/col root.

Reply:
> What kind of applications are expected for it outside this package?
This is only used in testing [here](https://github.com/lazyledger/lazyledger-core/blob/4464a087672f9ff4c5c6c6b0958f1b22064e7c45/types/block_test.go#L1362)

resolved https://github.com/lazyledger/lazyledger-core/pull/178/commits/d53622f01aa97dc3692b6962a882a9498ce5a996